spec/features/list_current_jobs_spec.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
```

Comment:
Why is this a feature spec and not a request spec?  A request spec is for a single request, whereas a feature spec typically crosses a number of requests.

Reply:
Because:
1. I'm testing the rendering of the interface (and not, say, an API).
2. It's less confusing to have one type of test rather than two, since (a) it is easier to find where something is tested and (2) there is only one DSL to use / keep straight, rather than two.

But that being said, agree that it could go either way. If you feel strongly, happy to switch.

Reply:
Request tests do render the interface see https://guides.rubyonrails.org/testing.html#implementing-an-integration-test. (Request specs are wrapping Rails integration tests)

You are not using the feature spec API (aside from visit).  The feature spec API is "fill_in ...", "click ...", etc.

Reply:
OK. Switched.